lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php

```diff
@@ -32,12 +33,22 @@ class GenerateCommand extends AbstractCommand
  */
 final class Version<version> extends AbstractMigration
 {
+    /**
+     * @param Schema $schema
+     *
+     * @throws AbortMigrationException
```

Comment:
This should be added if you abort the migration. By default, this doesn't throw an exception (see the empty method body).